packages/api/models/nomination.js

```diff
@@ -171,9 +162,10 @@ module.exports = (sequelize, DataTypes) => {
     },
     {
       hooks: {
-        beforeCreate: (nomination, option) => {
+        beforeCreate: async (nomination, option) => {
+          publicEmailDomains =  await sequelize.models.Domain.findAll();
```

Comment:
Instead of loading every value into memory and checking them in code, I think all this logic can be done as a single SQL query.

Reply:
Nevermind, this is in the beforeCreate hook so the approach I was thinking of won't work. This is fine as is since there probably won't be a ton of entries in the public email domain table.